### ISSUE TITLE
Re-enable disabled 8th GPU core on 8/7 M1 MBA late 2020?

### ISSUE THREAD

**Issue by Cobular** [AsahiLinux/m1n1#281]
Hey!

I was thinking the other day - would it be possible with a project like this to mess with the chip firmware and re-enable the factory disabled 8th core on the M1 late 2020 macbook air? Idk if they use e-fuses or something like that or not and the perf gain would probably be somewhat negligible.
I see they did sell a slightly better 8/8 CPU and GPU core model, but they have to have been a bit on the conservative side with the binning right? I feel like maybe the cores are still workable in some machines or something :P
Anyway this is probably really stupid but just something I thought about today at 2 am (I am very sleepy) and I think ya'll cool ppl would be the best bet to know if this is remotely possible.

If you have any idea, please let me know 🧡 !

TLDR: Download more cores when???

**Comment by marcan** [comment]
I think @asahilina will be able to comment more, but if it's anything like the rest of the fused out hardware, no, it's all locked and power-gated by the bootloader (or even hardware) before the OS runs. Firmware cannot be directly changed (and even if it could be hacked up via exploits/tricks in some cases, that is not something we're interested in supporting).
